addons/payment_transfer/models/payment.py

```diff
@@ -18,6 +18,35 @@ class TransferPaymentAcquirer(models.Model):
     def transfer_get_form_action_url(self):
         return '/payment/transfer/feedback'
 
+    def _format_transfer_data(self):
+        company_id = self.env.user.company_id.id
+        # filter only bank accounts marked as visible
+        journals = self.env['account.journal'].search([('type', '=', 'bank'), ('use_in_payment', '=', True), ('company_id', '=', company_id)])
+        accounts = journals.mapped('bank_account_id').name_get()
+        bank_title = _('Bank Accounts') if len(accounts) > 1 else _('Bank Account')
+        bank_accounts = ''.join(['<ul>'] + ['<li>%s</li>' % name for id, name in accounts] + ['</ul>'])
+        post_msg = '''<div>
+<h3>Please use the following transfer details</h3>
```

Comment:
@fpodoo perhaps it is better to indent this too so it follows the indentation of the function?